lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -2297,28 +2326,32 @@ ParseResult FIRStmtParser::parsePrintf() {
     auto constTrue =
         builder.create<ConstantOp>(UIntType::get(getContext(), 1), constOne);
     builder.create<CoverOp>(clock, condition, constTrue,
-                            builder.getStringAttr(formatStrUnescaped));
+                            builder.getStringAttr(formatStrUnescaped),
+                            builder.getStringAttr(""));
```

Comment:
Looking at these, I think usually there is a builder that takes a StringRef instead of a StringAttr, does this work?
```c++
    builder.create<CoverOp>(clock, condition, constTrue, formatStrUnescaped, "");
```
Here are the builders on my machine for CoverOp, not sure what they look like after adding another StringAttr param:
```c++
  static void build(::mlir::OpBuilder &odsBuilder, ::mlir::OperationState &odsState, ::mlir::Value clock, ::mlir::Value predicate, ::mlir::Value enable, ::mlir::StringAttr message);
  static void build(::mlir::OpBuilder &odsBuilder, ::mlir::OperationState &odsState, ::mlir::TypeRange resultTypes, ::mlir::Value clock, ::mlir::Value predicate, ::mlir::Value enable, ::mlir::StringAttr message);
  static void build(::mlir::OpBuilder &odsBuilder, ::mlir::OperationState &odsState, ::mlir::Value clock, ::mlir::Value predicate, ::mlir::Value enable, ::llvm::StringRef message);
  static void build(::mlir::OpBuilder &odsBuilder, ::mlir::OperationState &odsState, ::mlir::TypeRange resultTypes, ::mlir::Value clock, ::mlir::Value predicate, ::mlir::Value enable, ::llvm::StringRef message);
```

Reply:
Yep! This works. I just need to change the second to last argument to _not_ use a `StringAttr` and also use a `StringRef`. That's cleaner. :100: